src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -1104,7 +1101,7 @@ static TaskStatusHandler processCollectionResults(
                                     + " metrics.";
 
                     instanceStatus = Status.STATUS_WARNING;
-                    CustomLogger.laconic(LOGGER, Level.WARN, instanceMessage, 0);
+                    CustomLogger.laconic(LOGGER, instanceMessage, 0);
```

Comment:
Slf4j doesn't support passing in a log level as an argument, but warn was the only thing being passed in, so I just made that the default value.